BrainPortal/app/views/resource_usage/_resource_usage_table.html.erb

```diff
@@ -98,8 +99,54 @@
                      :value => (@total.to_f / @resource_usages.total_entries.to_f)
                     )
                   ) %>)
+  </p>
   <% end %>
 </fieldset>
+<fieldset>
```

Comment:
Please insert some blank lines before the `<fieldset>` starts and also, just liek for the previous fieldsets, add a nice `<p>` to separate them?